docs/options.md

```diff
@@ -635,7 +635,7 @@ To specify more than one environment variable, separate the variable names by sp
 ### `CIBW_BEFORE_ALL` {: #before-all}
 > Execute a shell command on the build system before any wheels are built.
 
-Shell command to prepare a common part of the project (e.g. build or install libraries which does not depend on the specific version of Python).
+Shell command that runs before creating Python environment, to build or install parts that do not depend on the specific version of Python.
```

Comment:
```suggestion
Shell command to prepare a common part of the project (e.g. build or install libraries which does not depend on the specific version of Python). This runs before creating the Python environments for each build.
```

Reply:
If  a "common part of the project" doesn't include pieces that are shared between operating system environments, it is better to leave this scope escape off to avoid bloating the description beyond that is mere necessary.